.storybook/main.js

```diff
@@ -25,6 +25,9 @@ const config = {
     name: "@storybook/web-components-vite",
     options: {},
   },
+  docs: {
+    autodocs: "tag",
```

Comment:
**Question:** At the beginning, I could not set this to `true` because `controller` and `template` docs were not recognized as "autogenerated" docs.

Setting this to `true` would have generated an auto doc in addition to our markdown doc that is supposed to replace it.

Since I've managed to make Storybook recognize our docs as true docs that may replace `autogenerated` docs, we can now use `true` here instead of manually adding `tags: ['autodocs']` in every story default export.

Still, I'm not sure this is a good idea because maybe one day we'll create a story that requires no doc?

Feel free to discuss this here and / or ping me to discuss it in sync.